docs/tools/documentation_generator.md

```diff
@@ -1,3 +1,6 @@
 ## Documentation Generator
 
-<!-- TODO(lucacasonto): write things -->
+`deno doc` followed by a list of source files will print the JSDoc documentation
+for each module's **exported** members.
+
+<!-- TODO(mattd3v): write more things, and add code examples -->
```

Comment:
In this same PR, I hope? No need to break it up to this extent.

Reply:
Yes, today I'll push the rest. Just wanted to put this here so others know it's being worked on!